packages/aragon-wrapper/src/core/proxy/index.js

```diff
@@ -12,6 +12,38 @@ export default class Proxy {
     this.initializationBlock = initializationBlock
   }
 
+  /**
+   * Fetches past events for a given block range
+   *
+   * @param {Array<String>} eventNames events to fetch
+   * @param {Object} options object with fromBlock and toBlock to specify the range
+   * @return {Observable} Single emission observable with the past events
+   */
+  pastEvents (eventNames, { fromBlock = this.initializationBlock, toBlock = null } = {}) {
+    // Get all events
+    if (!eventNames) {
+      eventNames = ['allEvents']
+    }
+
+    // Convert `eventNames` to an array in order to
+    // support `.events(name)` and `.events([a, b])`
+    if (!Array.isArray(eventNames)) {
+      eventNames = [eventNames]
+    }
+
+    if (eventNames.length === 1) {
+      //
+      return from(
+        this.contract.getPastEvents(eventNames[0], { fromBlock, toBlock })
+      )
+    } else {
+      // Get all events in the block range and filter
+      return from(
```

Comment:
Interesting, does `fromEvent()` not work here? I would've expected it to be completely the same as `.events()`.

Reply:
`getPastEvents` only returns a promise so fromEvent doesn't work. 

https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#getpastevents